contrib/google.golang.org/grpc/client.go

```diff
@@ -142,6 +148,9 @@ func doClientRequest(
 		cfg.clientServiceName(),
 		cfg.analyticsRate,
 	)
+
+	span.SetTag(ext.SpanKind, ext.SpanKindClient)
```

Comment:
Ideally we should only get one client span per outbound interaction, not sure if we want it both here and in the SendMsg part.